server/migrations/20190214005129-create-user-account.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('UserAccounts', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER
+            },
+            apiId: {
```

Comment:
whats this

Reply:
the postgres db at the mw layer is going to have an id field, and the apiId is meant to store the id as known by the API itself (in case there is any discrepancy), as this will ensure successful API calls on behalf of the user